### ISSUE TITLE
[Direct Line] Bot does not respond on the first message the user enters

### ISSUE THREAD

**Issue by brandmanager2017** [Microsoft/BotBuilder#3569]
## Bot Info 
* Bot handle: bm_hubbot  
* App ID:  9f2a90e0-b0db-43e2-8952-89f828accc3e
* SDK Platform: Node.js
* SDK Version: 3.9.0
* Active Channels: Direct Line, Facebook Messenger, Skype, Web Chat
* Deployment Environment: Azure App Service

## Issue Description
The bot does not respond on the first message that the user enters as a response the the bot's welcome message. 
This problem only occurs when I'm using Direct Line and the embedded Web Chat in https://dev.botframework.com it seems to be working fine when using the Emulator.

I also have tested using Messenger and Skype but I cannot replicate the issue since those channels does not have a restart function where I could be able to receive the bot's welcome message.

I have checked the logs from the emulator and there seems to be nothing strange there. Also checked the issues in https://dev.botframework.com, still no error is being logged. 

Can you help me figure out what's wrong with the bot?

**Comment by nwhitmont** [comment]
@brandmanager2017 are you still experiencing the issue after upgrading to the current `botbuilder@3.10.2`?

**Comment by brandmanager2017** [comment]
@nwhitmont Upgrading the botbuilder did not fix the issue. 
You can find the bot code here https://gist.github.com/brandmanager2017/7f4883b558c76a15be1442b1acc78a7d

Thanks!

**Comment by brandmanager2017** [comment]
@nwhitmont I have turned on logging services on my azure web app service to see the behavior of the app and I found out that every time I initialize a conversation with the bot using Direct Line it gives me this error https://gist.github.com/brandmanager2017/86d31728db269f9a3757e9dc3ecbd675.

This error is not thrown out when using the emulator.

I was also wondering why the direct line is calling https://BrandManagerBOT:80 when that is not the domain that I have used on the endpoint URL for the bot.
